src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -229,7 +229,13 @@ export const WalletOptions: Definitions<WalletConfig> = {
     conflicts: ["totalAccounts"]
   },
   totalAccounts: {
-    normalize,
+    normalize: rawInput => {
+      if (rawInput >= MAX_ACCOUNTS)
+        console.log(
+          `wallet.totalAccounts set above MAX_ACCOUNTS: ${MAX_ACCOUNTS} and may affect performance.`
```

Comment:
What about something like?

```suggestion
          `The wallet.totalAccounts value of ${rawInput} is greater than ${MAX_ACCOUNTS} which may affect performance.`
```

(if we change it to something like this then `rawInput >= MAX_ACCOUNTS` should change to `rawInput > MAX_ACCOUNTS`)